# Copyright (c) 2024. Tudor Oancea, Matteo Berthet
std_msgs/Header header
uint8 IDLE = 0
uint8 RACING = 1
uint8 NOMINAL_STOPPING = 2
uint8 EMERGENCY_STOPPING = 3
uint8 STOPPED = 4
uint8 state
